mne_bids/io.py

```diff
@@ -0,0 +1,43 @@
+from mne import io
+import os
+
+ALLOWED_EXTENSIONS = ['.con', '.sqd', '.fif', '.gz', '.pdf', '.ds']
+
+
+def _parse_ext(raw_fname):
+    """Split a filename into its name and extension."""
+    fname, ext = os.path.splitext(raw_fname)
+    # BTi data is the only file format that does not have a file extension
+    if ext == '':
```

Comment:
we should throw a warning here. Or some log message

Reply:
I can try to do this, though this code isn't an addition by me (I only repackaged it into a hidden function) and I'm not 100% sure of the specific use-case there...does BTi data just have no extension whatsoever?

Reply:
ahh the 2.7 problem is a string type check